### ISSUE TITLE
Replace the "Support" feature by "Probability vote" feature (useful in inductive arguments)

### ISSUE THREAD

**Issue by raindropsfromsky** [arguman/arguman.org#277]
This suggestion is related to  #270.

The "Support" vote count is not suitable for deductive reasoning, because if the logic is faulty, the viewer can always point out the fallacy  in it; or submit his own objection as a subnode. Just saying "I agree" does not make the logic stronger!

But this matters in Inductive logic. But here also, just supporting a premise does not make it logically stronger. This kind of argument will need an assessment of how likely the premise is.

And it is here that the problem described in #270 hits hard: The nodes in the top-left corner of the map get maximum eyeballs. Most users start here and may click "support" links in this corner of the map. But by the time they cross 10 nodes, their interest peters out, and they stop supporting the rest of the modes with equal enthusiasm. Thus the nodes in the top-left corner appear to be stronger than the rest.

To avoid this problem, instead of providing a "Support" link in every node, provide a "Probability vote" link. This will allow other viewers to assess the probability of the premise, on a 0-10 scale. 
(0= impossible, 10=100% certain)

By default, every premise will have a score of 10 (because the author believes it to be 100% true.)
But as more voters cast their votes, their average may reduce the overall score.

For example, if there are two voters with a score of 5 and 3, then 
Average score = (10+5+3)/3 = 6. 

But if the other two voters also support the premise fully (with scores of 10 each), the overall score will remain at 10. 

This feature has the following advantage over the "Support" feature:
 1 The score of any node is based on its merit; not by the actual number of voters.
 2 The nodes in the top-left corner of the map do not enjoy a favorable bias.
&nbsp;&nbsp;&nbsp;Thus all nodes are evaluated impartially.

**Comment by EHadoux** [comment]
In fact, in argumentation theory, the notion of support is a bit different (see, for instance, http://www-sop.inria.fr/members/Serena.Villata/Resources/comma10.pdf) 
As you mentioned in #270 it's more like an agreement vote. What you are proposing here is to multi-valuate this vote?

**Comment by raindropsfromsky** [comment]
Well, after posting that last message, I realized that just replacing the "Support" button with a 0-10 probability score will not help. 

- The probability score is valid only for [inductive](https://en.wikipedia.org/wiki/Inductive_reasoning) or [plausible](https://en.wikipedia.org/wiki/Plausible_reasoning) reasoning is used. 
- Arguments based on [deductive reasoning](https://en.wikipedia.org/wiki/Deductive_reasoning) should have neither the ***Support*** button nor the probability score; because it depends on logic only.
- In fact, in real life, most users are laymen, who do not have expertise in the subject under discussion. They try to apply [abductive reasoning](https://en.wikipedia.org/wiki/Deductive_reasoning). These can be attacked like deductive arguments.

To sum up, the "Support" button should be eliminated, and the "Probability rating" button should be provided only for inductive arguments. 

In other words, we need is a flag in each argument that identifies the type of reasoning, so that it can be attacked with proper technique.

Here, note that this flag is needed for each ***argument*** as a whole, not for each ***premise***.

But Arguman can handle only simple arguments (each node is an [independent premise](https://en.wikipedia.org/wiki/Argument_map#Key_features_of_an_argument_map)). As a result, it cannot demarcate a whole argument (see #275).
